src/WebJobs.Script/Diagnostics/OpenTelemetry/ResourceAttributeConstants.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics.OpenTelemetry
+{
+    internal sealed class ResourceAttributeConstants
+    {
+        internal const string AttributeCloudProvider = "cloud.provider";
```

Comment:
@RohitRanjanMS the host metric changes that also define these attributes:

https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Metrics/TelemetryAttributes.cs

And they are added in the meter for capturing host metrics:

https://github.com/Azure/azure-functions-host/blob/22849a0dc36d5c5d32cc93368ec95f51a5534737/src/WebJobs.Script/Metrics/HostMetrics.cs#L38-L57

We should ensure that all these changes are aligned.

Reply:
Sure, I will merge these attributes.